blackbox/build.gradle

```diff
@@ -68,6 +68,9 @@ task monitoringTest(type: Exec, dependsOn: ['bootstrap', 'unpackDistTar']) {
 
 task itest(type: Exec, dependsOn: ['bootstrap', 'unpackDistTar']) {
     commandLine "$projectDir/bin/test-docs"
+    if (project.hasProperty('file_name_filter')) {
```

Comment:
Is the property even required or could the env variable be set directly, or would gradle intercept it and not pass on to `test-docs`?

Reply:
The property is not required, you can export the env var in the terminal and then invoke the gradle task and it will be available for the python code. I have amended the code and changed the name of the property to ITEST_FILE_NAME_FILTER. I have also changed the commit message to reflect this.